src/errors.ts

```diff
@@ -0,0 +1,82 @@
+import { register } from "./core";
+import { instrument } from "./instrumentation";
+
+let regexEscape = (str) => str.replace(/([.?*+^$&[\]\\(){}|<>-])/g, "\\$1");
+
+export function logError(errorToLog) {
```

Comment:
Why are you exporting this function? Seems like it should be ErrorMonitor's internal routine